backend/DB/api/routes.py

```diff
@@ -298,30 +202,17 @@ def add_threshold():
     return jsonify([{'reward_type': threshold.reward_type, 'threshold': threshold.threshold}])
 
 
-@app.route('/api/threshold/get/<reward_type>', methods=['GET'])
-def get_threshold(reward_type):
-    threshold = queries.get_threshold(reward_type)
-    if threshold is None:
-        return jsonify([])
-    return jsonify([{'reward_type': threshold.reward_type, 'threshold': threshold.threshold}])
-
-
-@app.route('/api/threshold/update/<reward_type>', methods=['PUT'])
-def update_threshold(reward_type):
-    thres_json = request.get_json()
-
-    if thres_json is None or 'threshold' not in thres_json:
-        abort(400)
-
-    threshold = queries.get_threshold(reward_type)
-    threshold.threshold = thres_json['threshold']
-    queries.update_threshold(threshold)
-    return jsonify({'reward_type': threshold.reward_type, 'threshold': threshold.threshold})
+@app.route('/api/threshold/update/<reward_type>/<threshold>', methods=['PUT'])
+def update_threshold(reward_type, threshold):
+    thresh = queries.get_threshold(reward_type)
+    thresh.threshold = threshold
+    queries.update_threshold(thresh)
+    return jsonify({'reward_type': thresh.reward_type, 'threshold': thresh.threshold})
 
 
 def set_up_db():
     db.create_all()
-    update_users()
+    # update_users()
```

Comment:
Just remove unused code rather than commenting it out

Reply:
oops